src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,119 +33,188 @@ redirects:
 
-1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**.
-2. If applicable: From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select **Add a new app**.
-3. From the **Get started** page, select **Android** as the platform for mobile monitoring.
-4. Enter a name for your mobile app, and select **Continue**.
-
-Continue with the steps to [configure mobile monitoring](#configuration) with Gradle and Android Studio.
+<Steps>
+    <Step>
+        ### Add your Android app to New Relic
 
-OR:
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**.
+2. From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select **Add a new app**.
+3. From the **Get started** page, select **Android** as your mobile monitoring platform.
+4. Enter a name for your mobile app, then select **Continue**.
 
-* To complete the configuration process for a new mobile app later: Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**, then select **See instructions** next to your mobile app name.
-* To upgrade an existing Android installation: Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+    </Step>
+    <Step>
+        ### Update your Gradle build behavior through either the project or app build files
 
-## Configure with Gradle and Android Studio [#configuration]
+This configuration sets the project token, enables or disables features, and setting thresholds for alerts. You can configure one or both depending on your monitoring needs:
 
-These procedures to configure your Android app with Gradle and Android Studio also appear on the **Get started** page in New Relic.
+* To capture data across all apps in a project, configure on the project level. 
+* To capture data about one app within your Android project, configure on the app level. For example, let's say you have a build file for testing and a build file for production, but you only want to monitor your production build. Configure only the app build file to limit where the Android agent captures data. 
+* To set different configurations for different apps within a project, set a default configuration on the project level and app-level configurations on the app level.
 
-1. Merge New Relic's mobile monitoring code in the **Gradle & Android Studio** tab to your `build.gradle` file.
+Because the Android agent is not yet available as a community plugin, you need to provide the classpath through [MavenCentral](https://mvnrepository.com/artifact/com.newrelic.agent.android/agent-gradle-plugin) so the agent can monitor your Android app. 
 
-  <CollapserGroup>
+<CollapserGroup>
```

Comment:
Are we missing a step for people with multiple subprojects? I'm looking at the Gradle Plugin Portal step that has some users updating a kts file. @cthomas-newrelic